### PR TITLE
Revise information integrity goals

### PR DESCRIPTION
**Pull request by jbradleychen**
Acknowledge the importance of public information, and the role of the web in maintaining public information for the public good. Update language to replace terms and concepts that may become stale over time, and recognize importance of context and user trust decisions.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -243,12 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web should help people determine the trustworthiness of information
```
Comment (rhiaro) [inline index.html]:
```suggestion
The web must help people determine the trustworthiness of information, and make it possible for people to verify the information they see.
```

File: index.html [hunk 2]
```diff
@@ -243,12 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
   </h3>
   <p>
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
-The concept of origin and source is core to the web's security model.
+Society relies on the integrity of public information.
+As one of the most common ways people access information,
+the web must support the maintenance of public information for the public good
+at least as well as previous information platforms.
```
Comment (rhiaro) [inline index.html]:
```suggestion
The web must support the maintenance of public information for the public good.
```

File: index.html [hunk 3]
```diff
@@ -243,12 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web should help people determine the trustworthiness of information
   </h3>
   <p>
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
-The concept of origin and source is core to the web's security model.
+Society relies on the integrity of public information.
```
Comment (rhiaro) [inline index.html:249]:
```suggestion
Society relies on the integrity of public information.
We have a responsibility to build web technologies to counter misinformation.
```

File: index.html [hunk 4]
```diff
@@ -243,12 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
+As one of the most common ways people access information,
+the web must support the maintenance of public information for the public good
+at least as well as previous information platforms.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
```
Comment (rhiaro) [inline index.html:253]:
```suggestion
trustworthy web publishers and content.
The concept of origin and source is core to the web's security model.
```

**Review by rhiaro — CHANGES_REQUESTED** [review]
Discussing with @torgo @hadleybeeman and @maxpassion in our virtual face-to-face

File: index.html (hunk 1 above)
Comment (torgo) [reply index.html]:
```suggestion
The web must enable people to verify, and assess the trustworthiness of, information.
```

File: index.html [hunk 5]
```diff
@@ -243,11 +243,14 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web must enable people to verify, and assess the trustworthiness of, information.
```
Comment (torgo) [inline index.html]:
```suggestion
The web must make it possible for people to verify, and assess the trustworthiness of, information.
```

File: index.html [hunk 6]
```diff
@@ -243,11 +243,14 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
   </h3>
   <p>
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation.
+The web must support the maintenance of public information for the public good.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
```
Comment (rhiaro) [inline index.html]:
```suggestion
The public needs verifiable source and context information to recognize
trustworthy web publishers and content.
The web must support the maintenance of public information for the public good.
```

File: index.html [hunk 7]
```diff
@@ -243,11 +243,14 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
+The web must support the maintenance of public information for the public good.
 The concept of origin and source is core to the web's security model.
 We will make sure the new web technologies we create
 do not work against this architectural principle.
```
Comment (torgo) [inline index.html]:
```suggestion
The concepts of origin and source are core to the web's security model.
```

File: index.html [hunk 8]
```diff
@@ -243,11 +243,14 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
+The web must make it possible for people to verify, and assess the trustworthiness of, information.
   </h3>
   <p>
-We have a responsibility to build web technologies to counter misinformation,
-allowing information sources to be traceable and facts to be checkable.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
+The web must support the maintenance of public information for the public good.
```
Comment (rhiaro) [inline index.html]:
```suggestion
We have a responsibility to build web technologies to counter misinformation, 
and to maintain the integrity of information for public good.
The public needs verifiable source and context information to recognize
trustworthy web publishers and content.
```

**Review by hober — APPROVED** [review]

File: index.html [hunk 9]
```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
 
 <section>
   <h3 id="verify" data-export="" data-dfn-type="dfn">
-The web must make it possible for people to verify the information they see
+The web must make it possible for people to verify, and assess the trustworthiness of, information.
```
Comment (jyasskin) [inline index.html]: The commas here aren't quite right. I think we can just remove them?

```suggestion
The web must make it possible for people to verify and assess the trustworthiness of information.
```

File: index.html [hunk 10]
```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
-allowing information sources to be traceable and facts to be checkable.
-The concept of origin and source is core to the web's security model.
-We will make sure the new web technologies we create
-do not work against this architectural principle.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation, 
+and to maintain the integrity of information for public good.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
+The concepts of origin and source are core to the web's security model.
```
Comment (jyasskin) [inline index.html]: The [origin](https://html.spec.whatwg.org/multipage/browsers.html#origin) is certainly core to the security model, but I don't know of a technical definition of "source" or a place the security model relies on it. What's that part of this sentence talking about?

Comment (hadleybeeman) [reply index.html]: We have an issue open on that. We'd like to address that later, if we can. (But it does need addressing, we agree!) 
https://github.com/w3ctag/ethical-web-principles/issues/97

File: index.html (hunk 9 above)
Comment (rhiaro) [inline index.html]:
```suggestion
The web must make it possible for people to verify information and assess the trustworthiness of its source.
```

Comment (rhiaro) [reply index.html]: I think the meaning of the sentence is different with and without the commas (verifying information vs verifying trustworthiness of information). @hadleybeeman and I have proposed an alternative less clunky version below

File: index.html [hunk 11]
```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
```
Comment (rhiaro) [inline index.html]:
```suggestion
The concepts of [origin](https://html.spec.whatwg.org/multipage/browsers.html#origin) and [source](#transparent) are core to the web's security model.
```

File: index.html [hunk 12]
```diff
@@ -243,14 +243,15 @@ <h3 id="expression" data-export="" data-dfn-type="dfn">
...
-allowing information sources to be traceable and facts to be checkable.
-The concept of origin and source is core to the web's security model.
-We will make sure the new web technologies we create
-do not work against this architectural principle.
+Society relies on the integrity of public information.
+We have a responsibility to build web technologies to counter misinformation, 
+and to maintain the integrity of information for public good.
+The public needs verifiable source and context information to recognize
+trustworthy web publishers and content.
+The concepts of [origin](https://html.spec.whatwg.org/multipage/browsers.html#origin) and [source](#transparent) are core to the web's security model.
```
Comment (rhiaro) [inline index.html]:
```suggestion
The concept of origin and [its relationship information sources](https://www.w3.org/2001/tag/doc/distributed-content/) are core to the web's security model.
```

**Review by torgo — APPROVED** [review]
lgtm - with reference to tag finding on origin and source

**Review by rhiaro — APPROVED** [review]

**Pull request merged**